runtime/vm/visible.c

```diff
@@ -29,6 +29,62 @@
 #include "util_api.h"
 #include "vm_internal.h"
 
+
+
+/**
+ * Check module access from srcModule to destModule
+ *
+ * @param[in] currentThread the current J9VMThread
+ * @param[in] vm the javaVM
+ * @param[in] srcRomClass the accessing class
+ * @param[in] srcModule the module of the src class
+ * @param[in] destRomClass the accessing class
+ * @param[in] destModule the module of the dest class
+ * @param[in] destPackageID packageID of the dest class
+ * @param[in] lookupOptions J9_LOOK* options
+ *
+ * @return 	J9_VISIBILITY_ALLOWED if the access is allowed,
+ * 			J9_VISIBILITY_MODULE_READ_ACCESS_ERROR if module read access error occurred,
+ * 			J9_VISIBILITY_MODULE_PACKAGE_EXPORT_ERROR if module package access error
+ */
+VMINLINE IDATA
+checkModuleAccess (J9VMThread *currentThread, J9JavaVM* vm, J9ROMClass* srcRomClass, J9Module* srcModule, J9ROMClass* destRomClass, J9Module* destModule, UDATA destPackageID, UDATA lookupOptions)
+{
+	UDATA result = J9_VISIBILITY_ALLOWED;
+	if (srcModule != destModule) {
+		UDATA rc = ERRCODE_GENERAL_FAILURE;
+		if (!J9_ARE_ALL_BITS_SET(lookupOptions, J9_LOOK_REFLECT_CALL)) {
+			if (!isAllowedReadAccessToModule(currentThread, srcModule, destModule, &rc)) {
+				Trc_VM_checkVisibility_failed_with_errortypeRomClass(currentThread,
```

Comment:
Should this tracepoint report the `rc` value?

Reply:
the RC value is not necessary as the string last arg) reports the type of failure